src/main/java/hudson/plugins/parameterizedtrigger/BlockableBuildTriggerConfig.java

```diff
@@ -3,14 +3,9 @@
 import com.google.common.collect.ImmutableList;
 import hudson.Extension;
 import hudson.Launcher;
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
-import hudson.model.Action;
-import hudson.model.BuildListener;
+import hudson.model.*;
```

Comment:
No, the common practice in Jenkins is to refer all imports to prevent conflicts of new classes